### PR TITLE
adding synapse page

### PR DESCRIPTION
**Pull request by rimisra2**
Please make sure to merge this AFTER this PR is merged: https://github.com/microsoftgraph/microsoft-graph-docs/pull/18018

### CODE REVIEW

File: includes/data-connect-quickstart-08.md [hunk 1]
```diff
@@ -4,4 +4,284 @@ ms.localizationpriority: medium
...
+        ![A screenshot of the Visual Studio user interface showing the options to create a new project using ASP.NET Web Application.](../concepts/images/data-connect-vs-create-app.png)
+
+    1. Enter **EmailMetrics** for the name of the project.
+    1. Select **.NET Framework 4.7.2** for the framework option.
+    1. Select **Create**.
+
+    > [!IMPORTANT]
+    > Ensure that you enter the exact same name for the Visual Studio Project that is specified in this quick start instructions. The Visual Studio Project name becomes part of the namespace in the code. The code inside these instructions depends on the namespace matching the Visual Studio Project name specified in these instructions. If you use a different project name the code will not compile unless you adjust all the namespaces to match the Visual Studio Project name you enter when you create the project.
+
+    1. In the new **ASP.NET Web Application** project dialog, select MVC.
```
Comment (JarbasHorst) [inline includes/data-connect-quickstart-08.md:27]: Should this step be a sub node of "1. In the **New Project** dialog, do the following."? or should it be the step 3 in this sequence?

File: includes/data-connect-quickstart-06.md [hunk 2]
```diff
@@ -4,89 +4,153 @@ ms.localizationpriority: medium
...
+You can also use Azure Synapse instead of Azure Data Factory to create a pipeline to extract the data from Microsoft 365 to the Azure Storage account using Microsoft Graph Data Connect.
 
-## Approve consent requests
+## Create an Azure Synapse pipeline
 
-# [Microsoft 365 Admin Center](#tab/Microsoft365)
+1. Open a browser and go to your [Azure Portal](https://portal.azure.com/#home).
 
-1. Open a browser and go to your [Microsoft 365 Admin Portal](https://admin.microsoft.com).
+2. Sign in using an account with **Global administrator** rights to your Azure and Microsoft 365 tenants.
```
Comment (JarbasHorst) [inline includes/data-connect-quickstart-06.md:13]: In PR 18018, you have updated a similar instruction. Do you also want to update this sentence. Updated instruction based on PR 18018:

- Sign in using an account with **[Application Administrator](/azure/active-directory/roles/permissions-reference#application-administrator)** or **[Application Developer](/azure/active-directory/roles/permissions-reference#application-developer)** role to your Azure portal.

File: includes/data-connect-quickstart-06.md [hunk 3]
```diff
@@ -4,89 +4,153 @@ ms.localizationpriority: medium
...
+    6. **File system name:** flsynapse 
 
-1. In the **Data Access Request** call out, select the **Approve** button.
+        ![Screenshot shows create](../concepts/images/data-connect-synapse-create.png)
 
-    ![A screenshot showing a data access request pending consent approval in the Microsoft 365 admin center.](../concepts/images/data-connect-m365-approve.png)
-1. After a few moments, you should see the status page for the activity run update to show it is now _extracting data_.
+        ![Screenshot shows workspace](../concepts/images/data-connect-synapse-workspace.png)
 
-   <!-- ![A screenshot showing the Azure portal UI for the Data Factory service where the load status is now showing as "Extracting data".](../concepts/images/data-connect-adf-extraction-approved.png) -->
```
Comment (JarbasHorst) [inline includes/data-connect-quickstart-06.md:24]: I updated the following sentence to instruct the reader to navigate to the synapse workspace before launching Azure Synapse workspace. Please update this sentence if my instructions are wrong.

**Comment by JarbasHorst** [comment]
Hello @rimisra2, I reviewed your PR and left some comments. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PR [18292](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18292) into your branch to include my fixes and before addressing my comments.

**Comment by JarbasHorst** [comment]
Please, merge also this PR [18314](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18314) that includes fixes for the enumeration.

File: includes/data-connect-quickstart-06.md [hunk 4]
```diff
@@ -4,89 +4,153 @@ ms.localizationpriority: medium
 
 <!-- markdownlint-disable MD002 MD041 -->
 
-A Microsoft 365 administrator has the ability to approve or deny consent requests. This can be done via the Microsoft 365 Admin Center or programmatically via PowerShell. When you run a pipeline and trigger a PAM request, the request is attached to your user account that owns the service principal used by the pipeline. Even if the account is part of the approver group that you set up, you can't use it to approve the PAM request because self-approvals are not allowed. If you try, you'll get an error message in the PAM portal: "Requestor and approver are the same. Self-approval is not allowed." For development, you'll want to have a second account in addition to the admin who approves requests. Both the submitter and the approver must have active Exchange Online accounts.
```
[the remark this answers is not in the document]
Comment (rimisra2) [reply includes/data-connect-quickstart-06.md:7]: the adf set up and azure synapse set up are alternatives. We have customers that use either adf or synapse or want to set up both as they transition to synapse eventually.

File: includes/data-connect-quickstart-06.md [hunk 5]
```diff
@@ -4,89 +4,153 @@ ms.localizationpriority: medium
...
+4. Find the **Azure Synapse Analytics** resource type and use the following values to create it, then select **Create**. 
+    1. **Subscription:** select your Azure subscription 
+    2. **Resource group:** GraphDataConnect 
+    3. **Region:** pick an Azure region in the same region as your Microsoft 365 region 
+    4. **Name:** m365tostorage 
+    5. **Account name:** synapsedatalstorage 
+    6. **File system name:** flsynapse 
 
-1. In the **Data Access Request** call out, select the **Approve** button.
+        ![Screenshot shows create](../concepts/images/data-connect-synapse-create.png)
```
[the remark this answers is not in the document]
Comment (rimisra2) [reply includes/data-connect-quickstart-06.md:25]: done

File: includes/data-connect-quickstart-06.md [hunk 6]
```diff
@@ -4,89 +4,153 @@ ms.localizationpriority: medium
...
+
+4.	After you are in the **Activity runs** view, go to the **Activity runs** section, which is located in the bottom side of the page.
+
+5.	Hover over the **Activity name** and select the goggles option. This will bring up the **Details** tab.
+
+    ![Screenshot for activity runs](../concepts/images/data-connect-synapse-activity-run.png)
+
+6.	In the **Details** screen, look for the status of the pipeline activity as highlighted in the following image. In this case you can see it is in a state of **RequestingConsent**.
+
+    ![Screenshot for requesting consent](../concepts/images/data-connect-synapse-accept-request.png)
```
[the remark this answers is not in the document]
Comment (rimisra2) [reply includes/data-connect-quickstart-06.md:151]: done

File: includes/data-connect-quickstart-06.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (rimisra2) [reply includes/data-connect-quickstart-06.md:25]: done

File: includes/data-connect-quickstart-06.md (hunk 2 above)
Comment (rimisra2) [reply includes/data-connect-quickstart-06.md]: yes, thanks for the catch! I have changed it

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [18428](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18428) into this branch to include my final edits.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**